packages/nimble-components/src/patterns/button/styles.ts

```diff
@@ -139,39 +133,42 @@ export const styles = css`
     @layer active {
         .control:active {
             box-shadow: none;
+            color: ${buttonLabelFontColor};
```

Comment:
I'll point it out in one spot and let you update all the styles.ts files touched and recheck. Can you make sure these are in [DOM order based on the selector target](https://github.com/ni/nimble/blob/main/packages/nimble-components/docs/css-guidelines.md#group-selectors-by-target-and-document-order)? 

![image](https://github.com/ni/nimble/assets/1588923/3ab51fb5-0a43-4011-abd5-3fbf3e2bb0fc)


```
.control: active {}
.control:active::before {}
... [part start] {}
... [part end] {}
```